### ISSUE TITLE
Engagement Experience: Errors loading Onboard APEX Product and Create a Release Plan pages

### ISSUE THREAD

**Issue by tjprescott** [Azure/azure-sdk-tools#5271]
After opening the tool and choose the Azure Open AI Service, I try to navigate past the welcome screen and get the following errors:

![image](https://user-images.githubusercontent.com/5723682/215563013-fa755067-d3f2-4d6b-9836-ac52e450578b.png)

![image](https://user-images.githubusercontent.com/5723682/215562922-915adcca-e04a-4443-b07d-1b412e8da99f.png)

**Comment by ladonnaq** [comment]
@tjprescott This is related to this GitHub issue - https://github.com/Azure/azure-sdk-tools/issues/4264 and also this GitHub issue - https://github.com/Azure/azure-sdk-tools/issues/4537.  The current version of the release planner does not support non-APEX products or APEX products that are in GA.  

The behavior of Cloud Lifecycle is to “graduate” a product that **has met or has approved extensions** to the next lifecycle milestone.  Cloud Lifecycle (CLC) **does not track the completion of the extensions**.  So in CLC a product can be GA but still have extensions that they need to complete (which we are manually tracking now).  This is why OpenAI, Text Translator and MVAD are not showing up in the product dropdown box.  Cloud Lifecycle shows these products as GA and the current logic used to filter the products would see these as not in scope for APEX (because APEX is complete).  Wei is aware and is working on the issue #4264 to immediately unblock, while we determine best design for #4537.

**Comment by live1206** [comment]
Fixed and deployed in Prod:
![image](https://user-images.githubusercontent.com/5196139/215652527-0aed687f-0988-4b15-b39b-d2f5c5eab360.png)
![image](https://user-images.githubusercontent.com/5196139/215652615-12a553cd-6409-4020-aab2-44ecbca09471.png)
